CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing to Cobra
+
+Thank you so much for contributing to Cobra. We appreciate your time and help.
+Here are some guidelines to help you get started.
+
+## Code of Conduct
+
+Be kind and respectful to the members of the community. Take time to educate
+others who are seeking help. Harassment of any kind will not be tolerated.
+
+## Questions
+
+If you have questions regarding Cobra, feel free to ask it in the community
+[#cobra Slack channel][cobra-slack]
+
+## Filing a bug or feature
```

Comment:
Some of this could be remedied with an Issue and Feature template provided by Github.

Reply:
+1 Yes, I've found templates to be really useful. They can help issue authors give all the needed info, almost like a play book.